arrow-schema/src/datatype.rs

```diff
@@ -577,9 +577,14 @@ impl DataType {
         }
     }
 
-    /// Create a List DataType default name is "item"
+    /// Create a [`DataType::List`] where each elements has the
```

Comment:
Not sure 😄 but maybe?

```suggestion
    /// Create a [`DataType::List`] where each element has the
```